manifests/c/curlpipe/ox/0.7.6/curlpipe.ox.locale.en-US.yaml

```
@@ -0,0 +1,31 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: curlpipe.ox
+PackageVersion: 0.7.6
+PackageLocale: en-US
+Publisher: curlpipe
+PublisherUrl: https://github.com/curlpipe
+PublisherSupportUrl: https://github.com/curlpipe/ox/issues
+Author: curlpipe
+PackageName: Ox editor
+PackageUrl: https://github.com/curlpipe/ox
+License: GPL-2.0

```

```suggestion
License: GPL-2.0-or-later
```

is more suitable here? https://github.com/curlpipe/ox/blob/4a2ef2521c983d67c6aaf0796673a73791fdca95/LICENSE#L299